src/components/ContactCard/ContactCard.jsx

```diff
@@ -27,17 +32,32 @@ const ContactCard = ({ title, contact, renderHeader, renderBody }) => {
   const activeContactAccounts = getConnectedAccounts(contact)
 
   return (
-    <div>
+    <>
       {renderHeader(<ContactIdentity contact={contact} />)}
       {renderBody(
         <>
-          <ContactFields fields={normalizedFields} title={title} />
+          <Query query={allGroupsQuery}>
+            {({ data: allGroups, fetchStatus }) => {
+              if (fetchStatus === 'loaded') {
+                return (
+                  <ContactGroupsList
+                    contact={contact}
+                    allGroups={allGroups}
+                    title={t('contact_group')}
```

Comment:
Since we are displaying `t('contact_group')` even if the contact doesn't have any group, can we do something like : 

```jsx 
 {({ data: allGroups, fetchStatus }) => {
return (
<ContactGroupsList
contact={contact}
allGroups={allGroups}
title={t('contact_group')}
status={fetchStatus}
/>
```

And in the ContactGroupsList https://github.com/cozy/cozy-contacts/pull/483/files#diff-e89cc739ce1afe47f19587017466807bR18 if (status === 'loaded') { <ul>... } else { <Spinner /> } ? 

ATM : 
- the current displayed spinner is small, smaller than `contact_group` title. So on a slow network, we'll have a visual glitch 

- Are we sure about the "no group" status? Currently we'll display "Groups" with blanks and below `Contact Informations` : 
  - adding "no group assigned yet" can be a good solution?